tool/jt.rb

```diff
@@ -2171,6 +2171,9 @@ def graph(*args)
         graph = graphs.last
         raise "Could not find graph in #{dumps}" unless graph
 
+        FileUtils.cp graph, 'graph.bgv'
+        graph = 'graph.bgv'
```

Comment:
This assumes `jt` is run from the repo root (some code above does as well), I'll add `in_truffleruby_repo_root!` in this method